.github/workflows/workflow.yml

```diff
@@ -127,6 +127,31 @@ jobs:
           name: basilisk_runtime.compact.compressed.wasm
           path: target/release/wbuild/basilisk-runtime/basilisk_runtime.compact.compressed.wasm
 
+  build-benchmarks:
+    needs: pop-runner-instance
+    runs-on: ${{ needs.pop-runner-instance.outputs.runner }}
```

Comment:
this is not running in parallel, it just waits for runner to be free. you'll have to set it to github runner "ubuntu-latest" and you can remove need for pop-runner as you are nt using that runner anymore